lib/sqlalchemy/dialects/sqlite/base.py

```diff
@@ -1758,12 +1758,18 @@ def visit_create_index(
         return text
 
     def post_create_table(self, table):
-        text = ""
-        if table.dialect_options["sqlite"]["with_rowid"] is False:
-            text += "\n WITHOUT ROWID"
-        if table.dialect_options["sqlite"]["strict"] is True:
-            text += "\n STRICT"
-        return text
+        table_options = []
+
+        if not table.dialect_options["sqlite"]["with_rowid"]:
```

Comment:
I see the slight semantic change here however it looks like these both have boolean defaults in construct_arguments so it comes out the same, OK